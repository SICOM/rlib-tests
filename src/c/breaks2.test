COMMAND=./breaks2
FORMATS="txt"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
