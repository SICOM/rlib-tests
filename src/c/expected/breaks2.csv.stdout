Company,District,Region,Store,Sales     
,,,"Store #1                  ",$1500.00  
,,,"Store #2                  ",$1300.00  
,,,"Store #3                  ",$2000.00  
,,,"Store #4                  ",$1800.00  
,,,"Store #11                 ",$2500.00  
,,,"Store #12                 ",$2300.00  
,,,"Store #13                 ",$3000.00  
,,,"Store #14                 ",$2800.00  
